### PR TITLE
[KEP-3488]Promoting ValidatingAdmissionPolicy to GA

### PR DESCRIPTION
**Pull request by cici37**
Promoting ValidatingAdmissionPolicy to GA.
The code has been merged in 1.30. This is the doc updates.

### CODE REVIEW

**Comment by Princesso** [comment]
Hello @cici37  :wave: please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

**Comment by cici37** [comment]
I didn't assign any tech reviewer in this PR since it didn't touch any tech context. Please let me know if it's needed. Thanks

File: content/en/docs/reference/access-authn-authz/validating-admission-policy.md
Comment (sftim) [inline content/en/docs/reference/access-authn-authz/validating-admission-policy.md:1]: Extra credit: update the page to follow our [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) more closely

- headings in sentence case
- no backticks around API kinds like ValidatingAdmissionPolicy
- ValidatingAdmissionPolicy not Validating Admission Policy

That's a good first issue IMO (@cici37 - it definitely does not need to be you who does the work).

**Review by sftim — COMMENTED** [review]
Looks good to me

/lgtm
/approve

File: content/en/docs/reference/access-authn-authz/validating-admission-policy.md
Comment (cici37) [reply content/en/docs/reference/access-authn-authz/validating-admission-policy.md:1]: Thank you for the suggestion! We'll give another pass on the entire doc :)

Comment (cici37) [reply content/en/docs/reference/access-authn-authz/validating-admission-policy.md:1]: Thanks to @mjlshen who volunteered to work on the doc pass. Here is some suggestion for reference :)

**Pull request merged**